package-lock.json

```diff
@@ -3285,7 +3285,7 @@
         "animejs": "^2.2.0",
         "browserify-css": "^0.8.4",
         "custom-event-polyfill": "^1.0.6",
-        "debug": "github:ngokevin/debug#noTimestamp",
+        "debug": "github:ngokevin/debug#ef5f8e66d49ce8bc64c6f282c15f8b7164409e3a",
```

Comment:
This commit is the same one that was used by the branch pointer that was here previously. Wonder why it changed.